encryption_works.md

```diff
@@ -236,23 +242,25 @@ On the other end, the receiver uses PGP to decrypt the message and verify the e-
 
 PGP uses a public key and a private key, both strings of randomly-generated numbers and letters unique to each person. Information about the public key is stored on something called a public key server, which is a bit like a phonebook. When people want to e-mail you securely, they can look up your public key and send you an encrypted e-mail.
```

Comment:
Technically, they're two large prime numbers, not strings of random numbers and letters. They just look that way when they're ASCII-encoded to facilitate transport across various web protocols that don't do well with arbitrary binary data. 

I'm not sure if explaining it that way is more or less confusing...